src/styles/slider.css

```
@@ -0,0 +1,65 @@
+  }
+
+  & .slick-slide > div {
+    @apply px-5 xs:px-2.5 md:px-0;
+  }
+
+  & .slick-slide a,
+  & .slick-slide img,
+  & .slick-slide * {
+    user-select: none;
+    -webkit-user-drag: none;
+    -khtml-user-drag: none;
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)